static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -31,7 +31,7 @@ const tests = [
     acquisitionsEpicLiveBlog,
     acquisitionsEpicAlwaysAskElection,
     acquisitionsEpicThankYou,
-].map(Test => new Test());
+];
```

Comment:
with some of the tests in the array still in `javascripts-legacy` (and requiring `new`ing), will the rest of the file work when they're not instantiated?